pfs/lineLists/skyLines.txt

```diff
@@ -535,19 +535,20 @@
 681.84080            0.00    OH           1 UNKNOWN   8
```

Comment:
What is a "good set of problematic lines"? I can't make any sense of the commit message.